boards/FSM/Src/main.c

```diff
@@ -222,6 +224,9 @@ int main(void)
         Io_MSP3002K5P3N1_GetPressurePsi, MIN_BRAKE_PRESSURE_PSI,
         MAX_BRAKE_PRESSURE_PSI);
 
+    brake_actuation_status =
+        App_BinarySwitch_Create(Io_MSP3002K5P3N1_BrakeIsActuated);
```

Comment:
If we're `BinarySwitch` to shared, all the function name should be renamed to `App_SharedBinarySwitch_{method}`. Can you actually open a separate PR quickly just for that? I'll make sure to approve it quickly.

Reply:
Honestly, upon some more though. I don't think we should be using `BinarySwitch` here. The intent was to model a physical switch that have a very concrete definition of `ON` and `OFF`. Something like the brake status just doesn't have that. If you want, we can create a BinarySignal under `shared` instead with these methods:

```
App_DigitalInput_Create()
App_DigitalInput_Destroy()
App_DigitalInput_IsHigh()
```

Using the `IsTurnedOn` method of `BinarySwitch` seems a bit confusing imo.